src/modules/aws_account@0.0.1/migration/1644463885917-aws_account.ts

```diff
@@ -5,9 +5,31 @@ export class awsAccount1644463885917 implements MigrationInterface {
 
     public async up(queryRunner: QueryRunner): Promise<void> {
         await queryRunner.query(`CREATE TABLE "aws_account" ("id" SERIAL NOT NULL, "access_key_id" character varying NOT NULL, "secret_access_key" character varying NOT NULL, "region" character varying NOT NULL, CONSTRAINT "PK_9ad897024d8c36c541d7fe84084" PRIMARY KEY ("id"))`);
+        await queryRunner.query(`
+            create or replace procedure iasql_schedule_apply()
+            language plpgsql
+            as $$
+            begin
+                PERFORM graphile_worker.add_job('scheduleApply');
+                raise notice 'Scheduled apply';
+            end;
+            $$;
+        `);
+        await queryRunner.query(`
+            create or replace procedure iasql_schedule_sync()
+            language plpgsql
+            as $$
+            begin
+                PERFORM graphile_worker.add_job('scheduleSync');
+                raise notice 'Scheduled sync';
+            end;
+            $$;
+        `)
```

Comment:
once we have https://github.com/iasql/iasql-engine/issues/459, we can make add/replace for a blocking version that could calls `pg_sleep` and outputs something to `stdout`